go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/pingcap/goleveldb v0.0.0-20191226122134-f82aafb29989
 	github.com/pingcap/kvproto v0.0.0-20200228095611-2cf9a243b8d5
 	github.com/pingcap/log v0.0.0-20200117041106-d28c14d3b1cd
-	github.com/pingcap/parser v0.0.0-20200317021010-cd90cc2a7d87
+	github.com/pingcap/parser v0.0.0-20200325032611-b7d1e7e1b93d
```

Comment:
Any related parser PR?

Reply:
https://github.com/pingcap/parser/pull/786
And
https://github.com/pingcap/parser/pull/784